common/src/stack/report-system/command/report/system/tests/test_sles_patch.py

```diff
@@ -0,0 +1,124 @@
+import pytest
+import os
+import stack.api
+import hashlib
+import json
+
+
+# Returns hash of filename
+def get_hash(filename, sles):
+	# SLES15 uses sha256
+	if '15sp' in sles:
+		hasher = hashlib.sha256()
+	else:
+		hasher = hashlib.md5()
+
+	with open(filename, 'rb') as f:
+		hasher.update(f.read())
+
+	return hasher.hexdigest()
+
+
+# Verify SLES patches are installed
+def test_sles_pallet_patched(host, report_output):
+	total_matched = []
+	base_dir = '/export/stack/pallets/SLES/'
+	if not os.path.isdir(base_dir):
+		pytest.skip('No SLES pallet found - skipping patch check')
+	sles_flavors = os.listdir(base_dir)
+	assert sles_flavors
+
+	# Find out where is stack-sles*images*.rpm file(s)
+	result = host.run('find /export/stack/pallets/stacki/ -name "*stack-sles-*.rpm"')
```

Comment:
The pattern should have a `\` before the `.`:
```
"*stack-sles-*\.rpm"
```

Reply:
having `\` before `.` broke the code.